ios/ImagePickerManager.m

```diff
@@ -14,6 +14,12 @@ @interface ImagePickerManager ()
 
 @end
 
+@interface ImagePickerManager (UIImagePickerControllerDelegate) <UINavigationControllerDelegate, UIImagePickerControllerDelegate>
```

Comment:
The `UIImagePickerController`'s delegate is defined as `UIImagePickerControllerDelegate & UINavigationControllerDelegate` so it makes sense to put both into a single continuation